src/apps/dashboard/routes/users/add.tsx

```diff
@@ -197,6 +205,7 @@ const UserNew = () => {
                             type='text'
                             id='txtUsername'
                             label='LabelName'
+                            validator={{ pattern: '^([\\w \\-\'._@+]*)([\\w\\-\'._@+])([\\w \\-\'._@+]*)$', errMessage: 'Username must not be empty and contain only numbers, letters, spaces, or the following symbols -\'._@+' }}
```

Comment:
The text of the error message should be moved to `en-us.json` and `globalize.translate` should be used.

Reply:
Done, for both